cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -572,7 +572,7 @@ def _validate_cell_type_ontology_term_id(self):
         }
 
         def is_valid_row(row):
-            if row[cell_type_column] == "unknown":
+            if row[cell_type_column] == "unknown" or row[cell_type_column] == "na":
```

Comment:
this function does organism-specific logic to determine if the `cell_type_ontology_term_id` is valid. we can assume that `na` will be valid here because the tissue_type logic is implemented in schema_definition and isn't dependent on what type of organism the cell is.